client/src/app/site/motions/components/workflow-detail/workflow-detail.component.ts

```diff
@@ -96,7 +96,7 @@ export class WorkflowDetailComponent extends BaseViewComponent implements OnInit
      * Holds state permissions
      */
     private statePermissionsList = [
-        { name: 'Recommendation label', selector: 'recommendation_label', type: 'input' },
+        { name: this.translate.instant('Recommendation label'), selector: 'recommendation_label', type: 'input' },
```

Comment:
The view has the translation marked for this already.
See: https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/site/motions/components/workflow-detail/workflow-detail.component.html#L32

Reply:
No, we have to _mark_ the specific string to translate it. But `this.translate.instant` does not work as expected... (extract should put it in template-en.pot but it does not work for this strings...)

Reply:
Put in "marked translation" file and try to translate them in the view.

Reply:
That's not the best solution, because we have to maintain the string on two places - marked-translation is  used for mark-and-translate server-side strings - not for ts-strings. The extract should work with `this.translate.instant(..)`...

Reply:
`.instand` works synchronously. You would need to ensure that
1. The translations have been loaded before the site renders
2. That `.instant` was called again after the language changes.

That works fine for dialogs, pdf, exports and everything else that happens after you hit a button, but not for elements that should directly render on the view.